db-ops/job-create-db-postgres.yml.tpl

```diff
@@ -13,17 +13,10 @@ type: Opaque
 stringData:
   RDS_MASTER_PASSWORD: $MASTER_RDS_PASSWORD
   create-user.sql: |
-    SELECT 'CREATE DATABASE $DB_NAME' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '$DB_NAME');\gexec
-    SELECT 'REVOKE ALL PRIVILEGES ON DATABASE $DB_NAME FROM $DB_APP_USERNAME;' WHERE EXISTS (SELECT FROM pg_roles WHERE rolname = '$DB_APP_USERNAME');\gexec
-    SELECT 'DROP USER $DB_APP_USERNAME;' WHERE EXISTS (SELECT FROM pg_user WHERE usename = '$DB_APP_USERNAME');\gexec
-    CREATE USER $DB_APP_USERNAME WITH ENCRYPTED PASSWORD '$DB_APP_PASSWORD';
-    GRANT ALL PRIVILEGES ON DATABASE $DB_NAME TO $DB_APP_USERNAME;
-
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: $NAMESPACE
+    SELECT 'CREATE DATABASE __DB_NAME' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '__DB_NAME');\gexec
```

Comment:
What's with the __ here?

Reply:
it's a template format with __, modified to {{ }} for better understanding.